experimental/apps-mcp/lib/errors/errors.go

```diff
@@ -0,0 +1,262 @@
+// Package errors provides JSON-RPC 2.0 compliant error types for the MCP server.
+package errors
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// JSON-RPC 2.0 error codes as defined in the specification.
+const (
```

Comment:
Non-blocking note: this is good! we have an equivalent in `server.go` here https://github.com/databricks/cli/pull/3907/files?w=1#diff-3ee229a3a2abf9ff54e7405e515634100baecd96c94852f8985c156189a14524R49